.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "master"
+    ]
+}
```

Comment:
Please remove this file from your branch. 

To prevent this from hapening again, please add `.vscode` to the `.gitignore` file.

Reply:
Fixed, thanks for noticing!